src/index.html

```diff
@@ -64,27 +64,27 @@ <h2 class="headline__subtitle">LET’S CONNECT</h2>
         <ul class="social-stack">
```

Comment:
- [ ]
![l](https://user-images.githubusercontent.com/63972670/166486875-08a3f2bb-397f-4bfe-ba0d-4ecfb6ffa0bc.PNG)
 
- [ ] Please make sure It's possible to navigate through the page(Portfolio, About, Contact), focus on its elements.

Reply:
Dear reviewer, thank you for doing a good job. However, I am obliged to know if this particular change was part of the project requirements. Thank you